cogs/userinfo.py

```diff
@@ -0,0 +1,35 @@
+from discord.ext import commands
+import discord
+
+class useinfo(commands.Cog, name='Userinfo'):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(description = 'Information about a person.')
+    async def userinfo(self, ctx, member: discord.Member = None):
+        if not member:
+            member = ctx.author
+            embed=discord.Embed(title= f"User Information on @{member}", description=f'{member.mention}') 
+            embed.add_field(name= f"Name", value= member, inline=True)
+            embed.add_field(name='ID', value=member.id, inline=True)
+            embed.add_field(name='Nickname', value=member.nick, inline=True)
+            embed.add_field(name='Server', value=f'{ctx.guild.name}', inline=True)
+            embed.add_field(name='Status', value=member.status, inline=True)
```

Comment:
We don’t need the status and activity, however we need the account creation and join date